test/runtests.jl

```diff
@@ -95,11 +95,33 @@ end
 
         # compute optimal transport cost (Julia implementation + POT)
         c = sinkhorn2(μ, ν, C, eps; maxiter=5_000)
-        @test c isa Float32
+        @test c isa Vector{Float32}
```

Comment:
This is what I think we should avoid (IMO it's reasonable in PythonOT since that's what POT does :shrug: but in our implementation it would be nice to avoid it).